packages/video_player/video_player/lib/video_player.dart

```diff
@@ -314,8 +314,7 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
           _applyPlayPause();
           break;
         case VideoEventType.completed:
-          value = value.copyWith(isPlaying: false, position: value.duration);
-          _timer?.cancel();
+          pause().then((void pauseResult) => seekTo(value.duration));
```

Comment:
Is `void pauseResult` the right way to "[avoid anonymous parameter names](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-anonymous-parameter-names)"?

Reply:
Looks reasonable.